### DISCUSSION TITLE
Can the official release a web proxy server component?Thanks  so much!

### DISCUSSION THREAD

**Discussion by sqlnew** [dotnet/WatsonWebserver#132] [Ideas]
The Watson Web server is a great component that is very useful as an HTTP server. We hope to officially develop an HTTP (s) web proxy server component.

Previously, we had been using the "titanium web proxy" proxy server component successfully, but recently the author of this component has stopped developing it, making it impossible to upgrade in the future. Therefore, we strongly recommend that the official release of an HTTP proxy server with similar functions would be a very meaningful and great thing. Thank you for your outstanding work!!

**Comment by jchristn** [comment]
Hi @sqlnew thanks for the note.  I have another project called uscale (see https://github.com/jchristn/uscale) which uses WatsonWebserver under the hood.  It can be configured as a proxy or as a redirector, and can also load balance across nodes.  Please take a look and let me know what you think.  Moving this to discussion.
